services/economics-rewards/src/delegators.ts

```diff
@@ -0,0 +1,87 @@
+import { DelegationData, Delegator } from './types';
+import { getContractInstance } from './contract';
+import { saveDelegator } from './database';
+
+const getDelegations = async (
```

Comment:
What do you think about wrapping it into a `const` called `dposContractConnector` or something similar so that later on when called, it is clear that these methods fetch data from chain